core/chains/evm/txmgr/orm.go

```diff
@@ -228,10 +282,17 @@ func (o *orm) InsertEthTxAttempt(attempt *EthTxAttempt) error {
 }
 
 func (o *orm) InsertEthReceipt(receipt *EvmReceipt) error {
+	// convert to database representation
+	r := dbReceipt(*receipt)
```

Comment:
You have a potential panic here if receipt is nil. Why not pass in the value instead?

Reply:
That's a good point. InsertEthReceipt is only used for tests so the impact is minimal. But something we should def refactor (or add a nil check) for